lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
```

Comment:
small syntax thing, but normally we underscore method names in ruby instead of camelCase, ie:
`def calc_score`